platform/admin/core/admin-core-logviewer/src/main/webapp/js/backend/index.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+
+var es = require('event-stream')
+var concat = require('concat-stream')
+var http = require('http')
+
+var logs = []
+
+// retrieves logs from the endpoint
+var getLogs = function (done) {
```

Comment:
There shouldn't be a backend. Instead there should be get/success/failure logs actions and an action provider that encapsulates the AJAX side effect.

http://redux.js.org/docs/advanced/AsyncActions.html